src/app/components/shared/menu/button/button.component.ts

```diff
@@ -7,20 +7,24 @@ import { MenuAction } from "@interfaces/menusInterfaces";
 @Component({
   selector: "baw-menu-button",
   template: `
+    <!--
+      No tooltips for disabled buttons because of:
+      https://github.com/ng-bootstrap/ng-bootstrap/issues/1250#issuecomment-274916839
+    -->
```

Comment:
if this is needed, I believe bootstrap's solution is to use a wrapper element to which the tool tip is applied